### DISCUSSION TITLE
Chatbot Script Conflicts with Divi Theme CSS

### DISCUSSION THREAD

**Discussion by ccamedia** [FlowiseAI/Flowise#2996] [Q&A]
Hello everyone,

I'm currently integrating a custom chatbot on my WordPress site, which uses the Divi theme. The chatbot is based on Flowise and is embedded via a script in a Divi Code Module. While the chatbot works as expected, I'm encountering a significant issue: the CSS from the chatbot is conflicting with the CSS of the rest of my Divi site, leading to layout and styling problems.

Has anyone faced similar issues when integrating third-party scripts into a Divi-powered site? What’s the best way to isolate the chatbot’s CSS to prevent it from interfering with the global site styles? Any tips on how to ensure that the chatbot's styles are contained within its own div?

Thank you in advance for any advice or solutions you can offer!

Here you can see my test site for the bot: https://cca-media.at/test/

**Comment by sebastiangz** [comment]
Made a plugin and tested it in site with Divi latest version and worked. Check it out  https://github.com/sebastiangz/igis-flowise-bot, hope it works for you.
